memiavl/db_test.go

```diff
@@ -16,8 +16,9 @@ import (
 
 func TestRewriteSnapshot(t *testing.T) {
 	db, err := Load(t.TempDir(), Options{
-		CreateIfMissing: true,
-		InitialStores:   []string{"test"},
+		CreateIfMissing:     true,
+		InitialStores:       []string{"test"},
+		SnapshotWriterLimit: DefaultSnapshotWriterLimit,
```

Comment:
we can remove the default values in the test, because it'll be filled by default.